index.html

```diff
@@ -225,17 +233,19 @@ <h2 class="one">Skills</h2>
             </ul>
         </section>
 
-        <footer class="contact">
-            <h3>
-                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-            </h3>
-
-            <form action="https://formspree.io/f/xvolgewa" method="post" id="my-form">
-                <input type="text" placeholder="Full name" maxlength="30" name="full-name" required>
-                <input type="email" placeholder="Email address" name="email" required>
-                <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
-                <input type="submit" class="btn-green" value="Get in touch">
-            </form>
+        <footer class="footer-nav">
+            <div class="footer-grid">
+                <h3 class="paragraph">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h3>
+
+                <form action="https://formspree.io/f/xvolgewa" method="post" id="my-form">
+                    <input type="text" placeholder="Full name" maxlength="30" name="full-name" required>
+                    <input type="email" placeholder="Email address" name="email" required>
+                    <textarea name="message" placeholder="Typing something" maxlength="500" required></textarea>
+                    <input type="submit" class="btn-green" value="Get in touch">
+                </form>
+            </div>
         </footer>
```

Comment:
- [ ] Your footer is currently missing the social icons. Please add it to match the template.

![](https://i.imgur.com/4OjcBYM.png)